textproto/mod.ts

```diff
@@ -3,25 +3,42 @@
 // Copyright 2009 The Go Authors. All rights reserved.
 // Use of this source code is governed by a BSD-style
 // license that can be found in the LICENSE file.
-
 import type { BufReader, ReadLineResult } from "../io/buffer.ts";
 import { concat } from "../bytes/mod.ts";
-
 // Constants created for DRY
 const CHAR_SPACE: number = " ".charCodeAt(0);
 const CHAR_TAB: number = "\t".charCodeAt(0);
 const CHAR_COLON: number = ":".charCodeAt(0);
-
 const WHITESPACES: Array<number> = [CHAR_SPACE, CHAR_TAB];
-
 const decoder = new TextDecoder();
-
 // FROM https://github.com/denoland/deno/blob/b34628a26ab0187a827aa4ebe256e23178e25d39/cli/js/web/headers.ts#L9
 const invalidHeaderCharRegex = /[^\t\x20-\x7e\x80-\xff]/g;
 
-function str(buf: Uint8Array | null | undefined): string {
+export function str(buf: Uint8Array | null | undefined): string {
   return !buf ? "" : decoder.decode(buf);
 }
+export function parseCodeLine(
```

Comment:
Ditto. Do we need to export this?

I don't find this function in go's textproto module